src/sdam/events.ts

```diff
@@ -37,10 +49,12 @@ export class ServerDescriptionChangedEvent {
  * @category Event
  */
 export class ServerOpeningEvent {
-  /** A unique identifier for the topology */
+  /** A unique identifier for the topology  */
```

Comment:
Here and a bunch of the comments like it we have an extra space, can we clean up the whitespace changes?

Reply:
Just cleaned up the extra spaces!